pkg/client/didexchange/client.go

```diff
@@ -123,10 +149,34 @@ func (c *Client) CreateInvitation(label string) (*Invitation, error) {
 
 	sigPubKeyB58 := base58.Encode(sigPubKey)
 
-	// get the route configs
-	serviceEndpoint, routingKeys, err := mediator.GetRouterConfig(c.routeSvc, c.serviceEndpoint)
-	if err != nil {
-		return nil, fmt.Errorf("createInvitation: getRouterConfig: %w", err)
+	if opts.routerConnID == "" {
+		var connections []string
+
+		connections, err = c.routeSvc.GetConnections()
+		if err != nil {
+			return nil, fmt.Errorf("createInvitation: get connections: %w", err)
+		}
+
+		if len(connections) > 0 {
+			opts.routerConnID = connections[0]
```

Comment:
This default still makes no sense to me.

If the user has not provided a router connection ID, we shouldn't automatically take the first one.

Reply:
I thought it should be the default behavior. Like it was before.
@rolsonquadras @troyronda ^

Reply:
@llorllale 

> This default still makes no sense to me.

Any proposals/thoughts you have in mind for the default ?

Reply:
I think the user should be responsible for selecting the routers where they want to register their identifiers.

Barring that, next best option is to do randomized selection as default. My concern is mitigated, but not resolved, with a randomized approach.